### DISCUSSION TITLE
Unable to flash display on Ender 3 Max Neo

### DISCUSSION THREAD

**Discussion by jerome83136** [TheMDev/Ender_3_Max_Neo_4.2.2_Firmware#3] [General]
Hello,
I'm trying to flash my display with this firmware: https://img.staticdj.com/ea94dbbae15e04c278aa4309ede6ec55.zip?spm=..page_2412741.download_support_2.1

Followed instrutions from Creality website and from here https://github.com/TheMDev/Ender_3_Max_Neo_4.2.2_Firmware

I tried formating my SD card several times by using Windows and Linux. I also tried another SD-card.
I'm stuck, the update process never starts on the display after I power on my printer. (I never see the firmware update screen)

What did I do wrong ? Is there some workaround or this kind of issue ?

I folowed the same process for flashing the mainboard firmware and it worked without problem (with one of the SD cards I tried for the display)

Any help would be greatly appreciated ;-) 
Thank you for your help

Best regards

**Comment by Nightwalker813** [comment]
LCD on Ender 3 Max Neo may be 3 types:
DWIN
DACAI
TJC

Two of its have no problem with flashing. Many of versions can find in www
But TJC... It is very big problem for all, who want to use it with custom FW.
1. Most fw not have another langs except CN and EN(may be not in some fw).
2. Manually build fw can operate with it LCD only in simple interface (small text in left top side) without graphical icons and animations.
3. Most of FW in www for it LCD has problems with correct viewing. On LCD seen only indicates of temperature, position and speed of fan. Menu working, You can type button and select any submenus. But, You can`t see menu and your choise.
4. If You want to use Klipper - You may to disconnect LCD. It not working at all. Only LOGO - at all time.

I tried to find programs to edit fw. But it is has only to payment. And, may be, only in china lang. 
At some times ago, I found FW v.1.4.7_G on Creality Cloud. 10 Langs, but no choise to correct BLTouch position. I can`t use it, because I uses custom DUCT. My BLTouch trying to calibrate table at distance in air.
![IMG_20241113_220600836_HDR](https://github.com/user-attachments/assets/1679acd4-0672-442e-b4da-516d9cfc9b42)
 
If You want to use attached FW for TJC LCD on anothers fws - it will be work like in no.3 of my list.
